federatedrec/svd/README.md

```diff
@@ -0,0 +1,81 @@
+# Federated Singular Value Decomposition
+
+SVD is a  supervised learning approach that decompose a matrix into a product of matrices. It does matrix factorization via following formula:
+
+​								$r_{ui}=\mu+b_u+b_i+p_uq_i$
```

Comment:
It will be more intuitive if change the latex formula into images

Reply:
at the and of this file, is that svd instead of svd++?

Reply:
Thanks for point out. Latex formula replaced with images, and remove svd++ in readme.